app/code/Magento/Cms/Block/Block.php

```diff
@@ -10,6 +10,8 @@
 
 /**
  * Cms block content block
+ * @deprecated This class introduces caching issues and should no longer be used
```

Comment:
What issues?

Reply:
Must be this one, that was fixed? https://github.com/magento/magento2/commit/5c84c1ad6aecbf6209d04bcaa160c49f356344d4

Reply:
Not really. Except the fact the previous one uses both `id` and `identifier` for cache key, there was something more (I'll try to find out what was the other issue)